server/src/main/java/com/objectcomputing/checkins/services/kudos/KudosResponseDTO.java

```diff
@@ -0,0 +1,134 @@
+package com.objectcomputing.checkins.services.kudos;
+
+import com.objectcomputing.checkins.services.memberprofile.MemberProfile;
+import com.objectcomputing.checkins.services.team.Team;
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.core.annotation.Nullable;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+@Introspected
```

Comment:
Can use `@Getter` and `@Setter` to get rid of all the getter/setter boilerplate below

(as in `KudosCreateDTO`)